doc/rados/configuration/mon-config-ref.rst

```diff
@@ -600,6 +600,88 @@ is far outweighed by the number of accidental pool (and thus data) deletions it
 
 For more information about the pool flags see :ref:`Pool values <setpoolvalues>`.
 
+Monitor backup
+==============
+
+The monitor daemon is capable of creating consistant backups of the RocksDB to allow restorating
+in case of fatal multi node failure. It uses the native RocksDB backup functionality to create consistant 
```

Comment:
The Ceph Monitor can create consistent backups of its RocksDB to facilitate restoration in case the entire Monitor cluster fails.